src/ParallelAlgorithms/Initialization/MergeIntoDataBox.hpp

```diff
@@ -57,10 +57,7 @@ void merge_simple_tag_value(const gsl::not_null<db::DataBox<DbTagsList>*> box,
           << " that is already in the DataBox we found that the value being "
              "set by the action "
           << pretty_type::get_name<AddingAction>()
-          << " is not the same as what is already in the DataBox. The "
-             "value in the DataBox is: "
-          << db::get<SimpleTag>(*box) << " while the value being added is "
-          << simple_tag);
+          << " is not the same as what is already in the DataBox.");
```

Comment:
Please revert this change. You don't need this anymore once you have factored the action into a free function.

Reply:
The way I think we should handle this is:
```cpp
namespace db {
template <typename T>
std::string get_tag_value(const T& t) noexcept {
  return get_output(t);
}

template <typename T>
std::string get_tag_value(const std::unique_ptr<T>& t) noexcept {
  if (t == nullptr) {
    return "nullptr";
  }
  return get_output(*t);
}
}  // namespace db
```
in a file `DataStructures/DataBox/GetTagValue.hpp` and then invoke that function here as `db::get_tag_value(db::get<SimpleTag>(*box))`. I can put this together into a PR quick if you like, @geoffrey4444.

Reply:
OK, for now I'm going to leave this as-is, until we figure out what the real culprit is